priiloader/Makefile

```diff
@@ -121,15 +130,13 @@ ifeq ($(wildcard ../.git/.),)
 	@echo
 	@echo
 	@echo '#define GIT_REV 0x'$(NUMBER) > ../Shared/gitrev.h
-	@echo '#define GIT_REV_STR "0x'$(NUMBER)'-dirty"' >> ../Shared/gitrev.h
+	@echo '#define GIT_REV_STR "0x'$(NUMBER)'-no-git"' >> ../Shared/gitrev.h
 else ifeq ($(UNAME),Linux) 
-	@echo LINUX CURRENTLY UNSUPPORTED. PLEASE HELP ME MAKE A SCRIPT FOR LINUX
-	@exit 1
+	@echo "#define GIT_REV 0x$(shell git describe --always --long)" > ../Shared/gitrev.h
+	@echo "#define GIT_REV_STR \"0x$(shell git describe --always --long --dirty)\"" >> ../Shared/gitrev.h
 else
-#../Shared/SubWCRev.exe "../." "../Shared/svnrev_template.h" "../Shared/svnrev.h"
-#this exe just calls the bat file. starting bat from makefile was... eug
-#	-> system("call cmd /c ..\\Shared\\make_git.bat");
-		@../Shared/GitVersion.exe
+	@echo "#define GIT_REV 0x$(shell ../Shared/git.exe describe --always --long)" > ../Shared/gitrev.h
+	@echo "#define GIT_REV_STR \"0x$(shell ../Shared/git.exe describe --always --long --dirty)\"" >> ../Shared/gitrev.h
 endif

```

Comment:
the windows bat used to output the versions to the terminal , can the same be added after the endif ? (the bat had a line 'echo generated versions : %version% and %version_str%' . its just informational.

Reply:
Yeah, I can do that if you want. By the way, is there a reason you have the git.exe precompiled in the repository, wouldn't it make more sense to require git to be installed? 

https://git-scm.com/download/win 

Then there wouldn't be a need to have an (outdated) git client in the repository, and the code would be the same for Windows & Linux. 

If someone does have the .git folder, then he probably cloned the repo with git anyways and has it installed, no?

Reply:
that git is also part of the old b/s system. for svn it used to be required bla bla, yade yade yade.

we now generate a number if the source is not gotten through git, so that is no longer needed tbh. 

i just realised priiloader's build system is so so so so old and needs this cleanup xD

Reply:
Okay. I'll add both the version number output to the terminal, and stop using the git.exe in the repository.

Reply:
(so ye, that exe can be deleted i think)